roles/openshift_node/tasks/main.yml

```diff
@@ -98,6 +98,12 @@
       line: "OPTIONS=--loglevel={{ openshift.node.debug_level }}"
     - regex: '^CONFIG_FILE='
       line: "CONFIG_FILE={{ openshift_node_config_file }}"
+    - regex: '^HTTP_PROXY='
+      line: "HTTP_PROXY='{{ openshift.common.http_proxy }}''"
+    - regex: '^HTTPS_PROXY='
+      line: "HTTPS_PROXY='{{ openshift.common.https_proxy }}'"
+    - regex: '^NOPROXY='
+      line: "NOPROXY='{{ openshift.common.noproxy }}'"
```

Comment:
At this point, I'm wondering if we should just template /etc/sysconfig/{origin,atomic-openshift}-node